src/components/greetingSlice.js

```diff
@@ -0,0 +1,21 @@
+export const ACTIONS = {
+  GET_GREETING: 'GET_GREETING',
+};
+export const getGreeting = () => async (dispatch) => {
+  const json = await fetch('http://localhost:3001/greeting');
+
+  const GreetingsFromApi = await json.json();
+
+  dispatch({ type: ACTIONS.GET_GREETING, payload: GreetingsFromApi });
+
+  return GreetingsFromApi;
+};
```

Comment:
- [ ] The data from the backend cannot be fetched by the Front app. I looked at the dev tools console and discovered that was due to Cors policy blocking the local host. I recommend setting up your backend application so that all URLs can access the backend's data.

<img width="362" alt="image" src="https://user-images.githubusercontent.com/77190203/195984987-106e0b69-fc0f-47bd-9bef-45d1df18b3b6.png">